config/default/jenkins-release.yaml

```diff
@@ -91,6 +91,15 @@ jenkins:
                             privateKeySource:
                               directEntry:
                                 privateKey: ${SSH_RELEASE_PRIVKEY}
+                        - basicSSHUserPrivateKey:
+                            scope: GLOBAL
+                            id: "pkgserver"
+                            username: ${SSH_PKGSERVER_USERNAME}
```

Comment:
secrets not added yet?
https://github.com/jenkins-infra/charts-secrets/blob/master/config/release/jenkins/secrets.yaml#L1

Reply:
Better when I run git push, currently it's a none working ssh key, as  we don't want to accidently push any artifacts to pkg.jenkins.io, and I'll more than probably setup a ssh bastion to test the scripts